### PR TITLE
chore(doc): use valid example in doc

### PR DESCRIPTION
**Pull request by tearf001**
chained style not support yet
  

[Commit Convention]: https://www.notion.so/godialogue/Commit-Convention-84fd9a4c149e48c998d760f1c9176df0
[Code Review guidelines]: https://www.notion.so/godialogue/Code-Review-c5f3fcd185ca49aca73ade497c398fe9
[Draft PR]: https://github.blog/2019-02-14-introducing-draft-pull-requests

### CODE REVIEW

**Comment by tearf001** [comment]
At least 1 approving review is required by reviewers with write access.

**Comment by hadrien** [comment]
Thanks for your contribution and sorry for the delay, I was on vacation :-)

File: README.md [hunk 1]
```diff
@@ -90,7 +90,8 @@ def example(session: Session = Depends()):
 
 @router.get("/async_example")
 async def async_example(session: AsyncSession = Depends()):
-    return await session.execute("SELECT now()").scalar()
+    result = await session.execute("SELECT now()") # or juset return await session.scalar("SELECT now()")
+    return result.scalar()
```
Comment (hadrien) [inline README.md:94]:
```suggestion
    return await session.scalar("SELECT now()")
```

File: README.md [hunk 2]
```diff
@@ -121,7 +122,7 @@ def run_bg():
 
 async def run_async_bg():
     async with asyncio_support.open_session() as session:
-        await session.execute("SELECT now()").scalar()
+        await session.scalar("SELECT now()") # or async session.execute(...)
```
Comment (hadrien) [inline README.md:125]:
```suggestion
        await session.scalar("SELECT now()")
```

**Review by hadrien — APPROVED** [review]

**Pull request merged**